### ISSUE TITLE
[2.7.0] Widget "service monitoring" shows alert but host is deleted

### ISSUE THREAD

**Issue by joschi99** [Centreon-Widgets/centreon-widget-service-monitoring#32]
The service widget "service monitoring" reports a CRITICAL alert, but the host is deleted from the system.
![2015-12-23 11_49_50-centreon - it network monitoring](https://cloud.githubusercontent.com/assets/6400595/11975184/6701e542-a96b-11e5-93a4-5f5c6d0171d7.png)

**Comment by maksimatveev** [comment]
@joschi99 It helped me "Restart Monitoring Engine with Method Restart"
Apparently reload in some cases leaves a host or remote configuration services, as in my case.

**Comment by maksimatveev** [comment]
Yes, it is. Just checked, Method Restart solves this problem with a widget.

**Comment by maksimatveev** [comment]
Another problem with this widget is the message "You are not allowed to reach this page" when trying to open the host, with the services of such problems do not arise.
![07-01-16 18 31 49](https://cloud.githubusercontent.com/assets/15804154/12174359/25c81836-b56d-11e5-99d2-086ce24b5186.png)
![07-01-16 18 32 54](https://cloud.githubusercontent.com/assets/15804154/12174361/2a517834-b56d-11e5-9c53-e661e6e97d3c.png)

Maybe it's a separate issue, I do not know. Tested in Opera (Chrome), and Mozilla.

**Comment by lpinsivy** [comment]
Hi,

The first problem is not from the widget but from method to restart/reload Centreon Engine to update centreon_storage database.

The second problem is not yet present with Centreon web 2.7.3. Can you confirm ?

**Comment by maksimatveev** [comment]
@lpinsivy problems do not confirm to version 2.7.4.
